deeplearning4j-scaleout/spark/pom.xml

```diff
@@ -37,11 +37,33 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <spark.major.version>1</spark.major.version>
+        <spark.version>1.6.2</spark.version>
+        <hadoop.version>2.2.0</hadoop.version>
```

Comment:
There was hadoop.version 1 before which I thought was a bit ancient and I bumped it to the version spark 1.6/1.5 have